homeassistant/components/husqvarna_automower_ble/lawn_mower.py

```diff
@@ -0,0 +1,156 @@
+"""The Husqvarna Autoconnect Bluetooth lawn mower platform."""
+
+from __future__ import annotations
+
+import logging
+
+from homeassistant.components import bluetooth
+from homeassistant.components.lawn_mower import (
+    LawnMowerActivity,
+    LawnMowerEntity,
+    LawnMowerEntityFeature,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .const import DOMAIN
+from .coordinator import HusqvarnaAutomowerBleEntity, HusqvarnaCoordinator
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    config_entry: ConfigEntry,
+    async_add_entities: AddEntitiesCallback,
+) -> None:
+    """Set up AutomowerLawnMower integration from a config entry."""
+    coordinator: HusqvarnaCoordinator = hass.data[DOMAIN][config_entry.entry_id]
+    model = coordinator.model
+    address = coordinator.address
+
+    async_add_entities(
+        [
+            AutomowerLawnMower(
+                coordinator,
+                "automower" + str(model) + "_" + str(address),
```

Comment:
Also, for unique_id, address should be purely unique I assume? Reminder that unique_id is unique per platform per integration, so there's no need to prepend it with automower (if that's what you were trying to avoid)

Reply:
Yeah, address should be unique. I'll drop the other bits